bin/bridge-client/src/xpriv.rs

```diff
@@ -0,0 +1,51 @@
+//! Parses the operator's master xpriv from a file.
+
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+use bitcoin::bip32::Xpriv;
+use strata_key_derivation::operator::OperatorKeys;
+use strata_primitives::keys::ZeroizableXpriv;
+use zeroize::Zeroize;
+
+/// The environment variable that contains the operator's master [`Xpriv`].
+const OPXPRIV_ENVVAR: &str = "STRATA_OP_MASTER_XPRIV";
+
+/// Parses the master [`Xpriv`] from a file.
+pub(crate) fn parse_master_xpriv(path: &Path) -> anyhow::Result<OperatorKeys> {
+    let mut xpriv_str = std::fs::read_to_string(path)?;
+    let xpriv: ZeroizableXpriv = xpriv_str.parse::<Xpriv>()?.into();
+
+    // Zeroize the buffer
+    xpriv_str.zeroize();
+
+    OperatorKeys::new(&xpriv).map_err(|_| anyhow::anyhow!("invalid master xpriv"))
+}
+
+/// Resolves the master [`Xpriv`] from ENV vars or CLI.
+///
+/// If the `--master-xpriv` CLI argument is provided, it will be used.
+/// Otherwise, the `STRATA_OP_MASTER_XPRIV` ENV var will be used.
```

Comment:
This still seems incomplete, unless I'm misunderstanding something. I read it as the following:
- If a key is supplied via a CLI argument, it is used.
- Otherwise, if a file path is supplied, the key is read from the file.
- Otherwise, if the appropriate environment variable is set, the key is read from it.
- Otherwise, return an error.

Reply:
Done in 61c2e8b1448af4c96728b5eca4753262c47ca0e0